### DISCUSSION TITLE
Improve formatting of configuration directives in the documentation

### DISCUSSION THREAD

**Discussion by cuu508** [emersion/maddy#599] [Ideas]
Maddy's configuration directive docs are incredibly useful, but are not easy to scan visually. The reader must find the right line that starts with "Syntax:", then read the section, without "overshooting" into the next section:

![image](https://github.com/foxcpp/maddy/assets/661859/3da9db69-bdcf-441e-996a-300d27014dae)

I'm proposing a few formatting changes to make each section easier to find visually:

![image](https://github.com/foxcpp/maddy/assets/661859/6bbbfde9-c512-4276-8be8-61862e9c74f8)

I moved the "Syntax: " line to a `<h3>` element, and added `<hr>` separators between sections. I removed bold from the "Default:" line to reduce visual clutter. I made a tiny CSS change in the alb theme to make `<hr>` element thinner.

Markdown source (fragment):

```md
### debug _boolean_

Default: global directive value.

Enable verbose logging.

---

### domains _string list_

**Required**. <br>
Default: not specified.


ADministrative Management Domains (ADMDs) taking responsibility for messages.

Should be specified either as a directive or as an argument.

---
```

@foxcpp what are your thoughts on this? Yay or nay? I'd be happy to make a PR (or several smaller PRs) that reformat the docs.

**Comment by foxcpp** [comment]
That looks like a good improvement! PRs are welcome.

**Reply by foxcpp** [reply]
Sources for the theme are here: https://github.com/foxcpp/mkdocs_alb
